playbooks/roles/streisand-gateway/templates/instructions.md.j2

```diff
@@ -4,7 +4,7 @@
 {% endfor %}
 - - -
 
-Your Streisand Gateway contains step-by-step instructions for the services it provides, and mirrors of all necessary client software. The Gateway can be accessed over SSL or via a Tor Hidden Service.
+Your Streisand Gateway contains step-by-step instructions for the services it provides, and mirrors of all necessary client software. This document covers the SSL Certificate installation instructions and will show you how to connect to the Gateway to learn how to configure clients to use the Streisand services.
 
```

Comment:
This document covers the SSL Certificate installation instructions and will show you how to connect to the Gateway to learn how to configure clients to use the Streisand services -> Ce document couvre les instructions d'installation du certificat SSL et vous montrera comment vous connecter à la passerelle afin de décrire comment configurer vos clients pour se connecter aux services fournis par Streisand.